WeMo discovery crashes if there are no paired devices

**Reproduce:**
 * Install wemo binding
 * Connect WeMo Link with WiFi (do not add some LED light or something else)
 * Scan for Wemo -> WeMo Link should appear in Inbox
 * Approve WeMo Link (Bridge)
 * Execute Scan for "wemo"

**Log:**

```
2016-08-02 18:08:08.589 [TRACE] [b.w.d.WemoLinkDiscoveryService:109  ] - Starting WeMoEndDevice discovery on WeMo Link wemo:bridge:Bridge-1_0-231447B0100A09
2016-08-02 18:08:08.590 [TRACE] [b.w.d.WemoLinkDiscoveryService:113  ] - devUDN = 'uuid:Bridge-1_0-231447B0100A09'
2016-08-02 18:08:08.590 [DEBUG] [smarthome.io.net.http.HttpUtil:187  ] - About to execute 'http://192.168.0.121:49153/upnp/control/bridge1'
2016-08-02 18:08:08.606 [TRACE] [smarthome.io.net.http.HttpUtil:202  ] - <s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/"><s:Body>
<u:GetEndDevicesResponse xmlns:u="urn:Belkin:service:bridge:1">
<DeviceLists>0</DeviceLists>
</u:GetEndDevicesResponse>
</s:Body> </s:Envelope>
2016-08-02 18:08:08.607 [TRACE] [b.w.d.WemoLinkDiscoveryService:131  ] - endDeviceRequest answered '<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/"><s:Body>
<u:GetEndDevicesResponse xmlns:u="urn:Belkin:service:bridge:1">
<DeviceLists>0</DeviceLists>
</u:GetEndDevicesResponse>
</s:Body> </s:Envelope>'
2016-08-02 18:08:08.611 [ERROR] [b.w.d.WemoLinkDiscoveryService:207  ] - Failed to parse endDevices for bridge 'wemo:bridge:Bridge-1_0-231447B0100A09'org.xml.sax.SAXParseException: Content ist nicht zulässig in Prolog.
	at com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:257)
	at com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(DocumentBuilderImpl.java:339)
	at org.eclipse.smarthome.binding.wemo.discovery.WemoLinkDiscoveryService.startScan(WemoLinkDiscoveryService.java:144)
	at org.eclipse.smarthome.binding.wemo.discovery.WemoLinkDiscoveryService$WemoLinkScan.run(WemoLinkDiscoveryService.java:265)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Reason**
This is a correct response:
```
<s:Envelope s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/" xmlns:s="http://schemas.xmlsoap.org/soap/envelope/">
   <s:Body>
      <u:GetEndDevicesResponse xmlns:u="urn:Belkin:service:bridge:1">
         <DeviceLists><![CDATA[<?xml version="1.0" encoding="utf-8"?><DeviceLists><DeviceList><DeviceListType>Paired</DeviceListType><DeviceInfos><DeviceInfo><DeviceIndex>0</DeviceIndex><DeviceID>94103EA2B27898AD</DeviceID><FriendlyName>Lightbulb 01</FriendlyName><IconVersion>1</IconVersion><FirmwareVersion>83</FirmwareVersion><CapabilityIDs>10006,10008,30008,30009,3000A</CapabilityIDs><CurrentState>1,255:0,,,</CurrentState><Manufacturer>MRVL</Manufacturer><ModelCode>MZ100</ModelCode><productName>Lighting</productName><WeMoCertified>YES</WeMoCertified></DeviceInfo><DeviceInfo><DeviceIndex>1</DeviceIndex><DeviceID>94103EA2B2789877</DeviceID><FriendlyName>Lightbulb 02</FriendlyName><IconVersion>1</IconVersion><FirmwareVersion>7E</FirmwareVersion><CapabilityIDs>10006,10008,30008,30009,3000A</CapabilityIDs><CurrentState>0,135:0,,,</CurrentState><Manufacturer>MRVL</Manufacturer><ModelCode>MZ100</ModelCode><productName>Lighting</productName><WeMoCertified>YES</WeMoCertified></DeviceInfo></DeviceInfos></DeviceList></DeviceLists>]]></DeviceLists>
      </u:GetEndDevicesResponse>
   </s:Body>
</s:Envelope>
```

But  ```<DeviceLists>0</DeviceLists>``` cannot be parsed as xml in https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java#L144

I try to provide a fix and PR for this minor issue...

--- a/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java
+++ b/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java
@@ -45,7 +45,7 @@ import org.xml.sax.InputSource;
  * The {@link WemoLinkDiscoveryService} is responsible for discovering new and
  * removed WeMo devices connected to the WeMo Link Bridge.
  *
- * @author Hans-Jörg Merk - Initial contribution
+ * @author Hans-JÃ¶rg Merk - Initial contribution
  *
  */
 public class WemoLinkDiscoveryService extends AbstractDiscoveryService implements UpnpIOParticipant {
@@ -69,17 +69,24 @@ public class WemoLinkDiscoveryService extends AbstractDiscoveryService implement
      */
     private final static int SCAN_INTERVAL = 120;
 
+    /**
+     * The handler for WeMo Link bridge
+     */
     private WemoBridgeHandler wemoBridgeHandler;
 
     /**
      * Job which will do the background scanning
      */
     private WemoLinkScan scanningRunnable;
+    
     /**
      * Schedule for scanning
      */
     private ScheduledFuture<?> scanningJob;
 
+    /**
+     * The Upnp service
+     */
     private UpnpIOService service;
 
     public WemoLinkDiscoveryService(WemoBridgeHandler wemoBridgeHandler, UpnpIOService upnpIOService) {
@@ -135,7 +142,14 @@ public class WemoLinkDiscoveryService extends AbstractDiscoveryService implement
                                 "</DeviceLists>");
 
                         stringParser = StringEscapeUtils.unescapeXml(stringParser);
+                        
+                        // check if there are already paired devices with WeMo Link
+                        if("0".equals(stringParser)) {
+                        	logger.info("There are no devices connected with WeMo Link. Exit discovery");
+                            return;
+                        }
 
+                        // Build parser for received <DeviceList>
                         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
                         DocumentBuilder db = dbf.newDocumentBuilder();
                         InputSource is = new InputSource();

--- a/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java
+++ b/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java
@@ -145,7 +145,7 @@ public class WemoLinkDiscoveryService extends AbstractDiscoveryService implement
                         
                         // check if there are already paired devices with WeMo Link
                         if("0".equals(stringParser)) {
-                        	logger.info("There are no devices connected with WeMo Link. Exit discovery");
+                        	logger.debug("There are no devices connected with WeMo Link. Exit discovery");
                             return;
                         }
 

--- a/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java
+++ b/extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java
@@ -45,7 +45,7 @@ import org.xml.sax.InputSource;
  * The {@link WemoLinkDiscoveryService} is responsible for discovering new and
  * removed WeMo devices connected to the WeMo Link Bridge.
  *
- * @author Hans-JÃ¶rg Merk - Initial contribution
+ * @author Hans-Jörg Merk - Initial contribution
  *
  */
 public class WemoLinkDiscoveryService extends AbstractDiscoveryService implements UpnpIOParticipant {

